lib/OpenQA/Worker/Job.pm

```diff
@@ -36,6 +36,9 @@ has 'client';
 has 'developer_session_running';
 has 'upload_results_interval';
 
+#default list of files which would be uploaded from pool after job execution.
```

Comment:
* space after `#`
* s/would be/will be/

Reply:
done